### ISSUE TITLE
A confusing question about the inference

### ISSUE THREAD

**Issue by Lecxxx** [liuqk3/PUT#16]
Hi,
Thanks for your contribution,but I'm confused about the inference.
I must run the Pvqvae inference first, 

`python scripts/inference.py --name OUTPUT/pvqvae_exp_name/checkpoint/last.pth --func inference_reconstruction --gpu 0 --batch_size 8`

then run the transformer inference second

`python scripts/inference.py --name OUTPUT/transformer_exp_name/checkpoint/last.pth --func inference_complet_sample_in_feature_for_evaluation --gpu 0 --batch_size 1`

Is it right?

**Comment by liuqk3** [comment]
No. Just 
```
python scripts/inference.py --name OUTPUT/transformer_exp_name/checkpoint/last.pth --func inference_complet_sample_in_feature_for_evaluation --gpu 0 --batch_size 1
```
is enough.

**Comment by Lecxxx** [comment]
@liuqk3 Thanks for the reply.
But after I run 
`python scripts/inference.py --name OUTPUT/transformer_naturalscene/checkpoint/000199e_495399iter.pth --func inference_complet_sample_in_feature_for_evaluation --gpu 1 --batch_size 1`
the completed rusult is terrible.
![Places365_val_00033355](https://user-images.githubusercontent.com/49981610/204140899-39bfe749-a306-482b-9c98-4ce04ac358ac.png)
![Places365_val_00030843](https://user-images.githubusercontent.com/49981610/204140905-026faf5e-2ca3-4ccc-ba0a-50558c4c8df1.png)

**Comment by liuqk3** [comment]
@Lecxxx ,

It seems that the model is not load correctly. You can have a look at this.

**Comment by Lecxxx** [comment]
@liuqk3 
hi
Thank you for your reply, but I haven't seen the solution you provided, and I would like to ask if the training can be carried out on multiple GPUs, and how to set parameters if possible

**Comment by Lecxxx** [comment]
@liuqk3 please reply,thanks.

**Comment by Lecxxx** [comment]
@liuqk3 
Thanks for the reply.
In the prepare_data.md,I found that you only used some pictures of val in Places2,Could you tell me if the Inference part only uses 800 photos in places2?

**Comment by liuqk3** [comment]
@Lecxxx ,
Yes. We follow ICT, the testing images on FFHQ, Places2 and ImageNet are exactly the same with ICT.
Only 800 images are used for inference to get the metrics.

**Comment by Lecxxx** [comment]
@liuqk3 
I also found that you used Places2_ Natural is part of the data from Place365-Challenge-2016. I can't get Place365-Challenge-2016 correctly because the download address you provided has expired. Could you please provide a reliable download source?

**Comment by Hgy12345** [comment]
> @liuqk3 Thanks for the reply. But after I run the completed rusult is terrible.`python scripts/inference.py --name OUTPUT/transformer_naturalscene/checkpoint/000199e_495399iter.pth --func inference_complet_sample_in_feature_for_evaluation --gpu 1 --batch_size 1` ![Places365_val_00033355](https://user-images.githubusercontent.com/49981610/204140899-39bfe749-a306-482b-9c98-4ce04ac358ac.png) ![Places365_val_00030843](https://user-images.githubusercontent.com/49981610/204140905-026faf5e-2ca3-4ccc-ba0a-50558c4c8df1.png)

Have you solved the problem? I had a similar situation
![image](https://github.com/liuqk3/PUT/assets/106374654/68175945-e547-40ea-beeb-98d97b1832c6)
